test.disabled/ejabberd_tests/tests/rest_client_SUITE.erl

```diff
@@ -439,12 +453,19 @@ get_room_messages(Client, RoomID, Count, Before) ->
     Path = erlang:iolist_to_binary(PathList),
     get_messages(Path, Creds).
 
-create_room({_AliceJID, _} = Creds, RoomID, Subject) ->
-    Room = #{name => RoomID,
+create_room({_AliceJID, _} = Creds, RoomName, Subject) ->
+    Room = #{name => RoomName,
              subject => Subject},
     {{<<"200">>, <<"OK">>}, {Result}} = rest_helper:post(<<"/rooms">>, Room, Creds),
     proplists:get_value(<<"id">>, Result).
 
+update_room({_AliceJID, _} = Creds, RoomName, Subject, RoomID) ->
```

Comment:
According to REST's definition for `PUT` method it can be used to created or update an item. If that's what you are asking about.